WebRoot/messages/ZaMsg_fr.properties

```
@@ -1188,6 +1188,7 @@ NAD_ForwardTo = Renvoyer mail \u00e0
 NAD_COSName = Nom de la classe COS
 NAD_PwdLocked = Emp\u00eacher l\u2019utilisateur de modifier le mot de passe
 NAD_AllowUsernameInPassword = Autoriser le nom d\u0027utilisateur dans le mot de passe
+NAD_PasswordExpiryReminder = Rappel d'expiration du mot de passe
```

`'` should work, but it is usually replaced with `\u2019`

```suggestion
NAD_PasswordExpiryReminder = Rappel d\u2019expiration du mot de passe
```

---

Done